.github/workflows/test-performance.yml

```
@@ -57,7 +57,54 @@ on:
                 - 'Test/Altinn.Correspondence.LoadTests/correspondence/get-correspondence.js'
                 - 'Test/Altinn.Correspondence.LoadTests/correspondence/get-correspondence-overview.js'
                 - 'Test/Altinn.Correspondence.LoadTests/correspondence/get-legacy-correspondences.js'
-          
+         
+    workflow_call:
+        inputs:
+            apiVersion:
+                description: 'API Version'
+                required: true
+                type: string
+                default: 'v1'
```

actionlint (1.7.7)

input "apiVersion" of workflow_call event has the default value "v1", but it is also required. if an input is marked as required, its default value will never be used

(events)